packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -828,8 +831,9 @@ async function processRequest(params) {
     status: 'Pending',
     location: buildS3Uri(systemBucket, reportKey),
   };
-  await reconciliationReportModel.create(reportRecord);
-  log.info(`Report added to database as pending: ${JSON.stringify(reportRecord)}.`);
+  let apiRecord = await reconciliationReportModel.create(reportRecord);
+  await indexReconciliationReport(esClient, apiRecord, process.env.ES_INDEX);
```

Comment:
What do you think about using this function? https://github.com/nasa/cumulus/blob/master/packages/api/bin/serveUtils.js#L232

Reply:
The `addReconciliationReports` function is for localApi testing, dashboard uses it.

Reply:
Ah okay good to know!